pkg/apis/garden/validation/validation.go

```diff
@@ -492,7 +578,35 @@ func validateAlicloudVolumeTypeConstraints(volumeTypes []garden.AlicloudVolumeTy
 		}
 	}
 
-	allErrs = append(allErrs, validateVolumeTypeConstraints(types, fldPath)...)
+	allErrs = append(allErrs, validateVolumeTypes(types, fldPath)...)
+
+	return allErrs
+}
+
+func validateRegions(regions []garden.Region, fldPath *field.Path) field.ErrorList {
```

Comment:
Should we add also validation the prevents regions with duplicating names? Currently you can have
```yaml
  - name: us-east-2
    zones:
    - name: us-east-2a
  - name: us-east-2
    zones:
    - name: bar
```

Reply:
Yes, we can do it 👍 but let's do it outside of this PR. This validation feature wasn't there before, and it's not critical to continue because the `CloudProfile`s are controlled by operators. Can you create an issue?